pgcli/main.py

```diff
@@ -823,7 +823,7 @@ def cli(database, username_opt, host, port, prompt_passwd, never_prompt,
 
     if dsn is not '':
         try:
-            cfg = load_config(config_full_path)
+            cfg = load_config(pgclirc, config_full_path)
```

Comment:
I think you have the order of the arguments reversed. The second arg is the def_cfg which should point to pgclirc.